exonum-java-binding-core/rust/integration_tests/tests/service_proxy.rs

```diff
@@ -71,6 +72,7 @@ fn service_id() {
 }
 
 #[test]
+#[ignore]
```

Comment:
Can we ignore the whole test suite (a single ignore per file)?

Reply:
No. One of the possible solutions is to disable the whole module, but then it will not be even compiled and now we have a way to run these tests locally (with --ignored flag)